### ISSUE TITLE
Make configurable per repo

### ISSUE THREAD

**Issue by meaku** [peerigon/probot-always-assign#2]
It should be possible to enable / disable `always-assign` on a per repo basis.

**Comment by leomelzer** [comment]
This is probably the way to do it:

- https://andrewlock.net/creating-my-first-github-app-with-probot-part-2-creating-the-auto-assign-issues-bot/
- https://probot.github.io/api/latest/classes/context.html#config

```ts
    const config: AppConfig = await context.config<AppConfig>(ConfigFilename, { addAssignees: true })

    if(!config.addAssignees){
      return;
    }

```

**Comment by jnnkky** [comment]
@meaku Should always assign be enabled by default? or Disabled? Seems easier to enable by default and disable where needed. Whats your plan here? Were you going to enable it for more repos than you would disable it for? If so, I'd set it enabled by default and make it so you can disable via the config.

**Comment by leomelzer** [comment]
I'd propose to use the exact snippet like I copied above -- provide a default config to `context.config` which **enables** it. Then we can selectively disable. Opinionated defaults 👌

**Comment by leomelzer** [comment]
- Closed by #3
- Documented in 92a9607
- Used already here: https://github.com/peerigon/newww/blob/master/.github/always-assign.yml


To Deploy Peeribot you'll need to

- clone the repo 
- run `npm install --save probot-always-assign`
- `package-lock.json` gets updated
- Commit
- Create PR
- Merge
- Jenkins rebuilds peeribot

**Comment by meaku** [comment]
Awesome! Thanks @jannikkeye @leomelzer ❤️
